cluster/common.sh

```diff
@@ -656,7 +656,9 @@ NON_MASQUERADE_CIDR: $(yaml-quote ${NON_MASQUERADE_CIDR:-})
 KUBE_UID: $(yaml-quote ${KUBE_UID:-})
 ENABLE_DEFAULT_STORAGE_CLASS: $(yaml-quote ${ENABLE_DEFAULT_STORAGE_CLASS:-})
 ENABLE_APISERVER_BASIC_AUDIT: $(yaml-quote ${ENABLE_APISERVER_BASIC_AUDIT:-})
+ENABLE_APISERVER_ADVANCED_AUDIT: $(yaml-quote ${ENABLE_APISERVER_ADVANCED_AUDIT:-})
 ENABLE_CACHE_MUTATION_DETECTOR: $(yaml-quote ${ENABLE_CACHE_MUTATION_DETECTOR:-false})
+ADVANCED_AUDIT_BACKEND: $(yaml-quote ${ADVANCED_AUDIT_BACKEND:-log})
```

Comment:
From chat history:

> timstclair: So how come the other webhook URLs aren't included there? (e.g. GCP_AUTHN_URL, GCP_AUTHZ_URL)
> mikedanese: cause we only set those in kube-env's built from the gke apiserver
so there's no need to put them in opensource
or allow them to be toggled in opensource
configure-helper.sh still has to detect their existance in opensource but we don't have to plumb them through kube-up.sh

So we can consider this resolved.